thesis-de/content-de.tex

```diff
@@ -360,6 +362,128 @@ \subsubsection{Durchführung der Evaluierung}
+  Entsprechend soll die Software nur eine leere Hülle mit Standartfunktionen
+  einer API zur Verfügung stellen. Eine Konfiguration von einzelnen Parametern
+  ist dabei erwünscht. Zu diesem Parametern gehört als Beispiel der Port, auf
+  welchem das API-System betrieben und von anderen Systemen erreicht werden
+  kann. Zur Hilfe der admnistration soll das API-System ein Framework für
+  Entwickler bereitstellen. Mit diesem können wir als Projektmitglieder
+  anschließend die Logik in das System implementieren und somit definieren
+  welche Funktionen die API ausüben darf. Ein Framework in einer bekannten
+  Entwicklersprache ist dabei erwünscht. Dies hat den Vorteil, dass der 
+  zuständige Entwickler im Projekt keine weitere Einarbeitungszeit für die
+  neue Entwicklersprache benötigt.
+  \1 Das API-System sollte innerhalb des Projektes keine Kosten verursachen.
+  Dazu gehören Lizenzkosten sowie laufende Betriebskosten bei externen 
+  Cloudanbietern. Spätere Kosten bei dem Auftraggeber sind in dieser 
+  Anforderung nicht mit inbegriffen.
+  \1 Die Erstellung von späteren Anwendungshandbüchern der API, in Fachkreisen
+  auch API-Dokumentation genannt, sollte dem API-Entwickler so einfach wie
+  möglich fallen. Eine sehr beliebte Dokumentationsmöglichkeit ist dabei das
+  dokumentieren von wichtigen Informationen direkt in dem Programmiercode des
+  API-Systems. Entwickler können dabei über die Kommentierungs-Funktion der 
+  jeweiligen Entwicklersprache eine Beschreibung sowie Ein- und Ausgabe von
+  Werten festlegen. Diese werden im Anschluss während dem Kompiliervorgang
+  in einem struktierten Dokument festgehalten und neben der eigentlichen
+  API-Software mit ausgeliefert. Dieses seperate Dokument ist oftmals in 
+  Form einer HTML oder PDF und kann auf gängigen Computersystemen eingesehen
+  werden. Die beschriebene oder ähnliche Dokumentationsmöglichkeit sollte die
+  API-Software zur Verfügung stellen.
+  \1 Bereits vorhandenes Wissen über das API-System. Dazu gehört, dass diese
+  eine umfangreiche Dokumentation besitzt und eine aktive und große Community
+  für Diskussionen und Fragen vorhanden ist. Die dazugehörige
+  Programmiersprache der API sollte ebenfalls für die Projektmitglieder
+  nachvollziehbar sein, sowie eine umfangreiche Dokumentation und aktive
+  Community für Diskussionen und Fragen besitzen.
+\end{outline}
+\nl%
+
+\subsubsection{Durchführung der Evaluierung}
+\label{subsubsec:api_durchfuehrung_der_evaluierung}
+Bei der Evaluierung der API-Systeme wurden die in Punkt
+~\ref{subsubsec:api_vorbereitung_der_evaluierung} festgehaltenen Anforderungen
+der API als Leitlinie für die Durchführung verwendet. Als ersten Schritt wurde
+zunächst Recherche betrieben, damit das Projektteam einen allgemeinen Eindruck
+über die aktuelle Marktsitutation zu API-Systemen erhält. Dabei wurden im
+Internet mehrere Blogs, Foren sowie Artiek von Fachzeitschriften quer gelesen.
```

Comment:
teilweise fixed in https://github.com/DCC-org/documents/pull/68/commits/163c9d035ece91d491ebed273b49ae7bb2028366